CHANGELOG

```diff
@@ -1,6 +1,6 @@
 * 2.12.3 (2019-XX-XX)
 
- * n/a
+ * fixed number formatter in Intl extra extension
```

Comment:
I would add `when using a formatter prototype`, as the fix is only in this case.

Reply:
done